src/lib.rs

```diff
@@ -681,6 +700,198 @@ impl<'a> Default for LocalExecutor<'a> {
     }
 }
 
+/// A leaked async [`Executor`].
+///
+/// Largely equivalent to calling `Box::leak(Box::new(executor))`, but spawning, running, and
+/// finishing tasks are optimized with the assumption that the executor will never be `Drop`'ed.
+/// A leaked executor may require signficantly less overhead in both single-threaded and
+/// mulitthreaded use cases.
+///
+/// As this type does not implement `Drop`, losing the handle to the executor or failing
+/// to consistently drive the executor with [`tick`] or [`run`] will cause the all spawned
+/// tasks to permanently leak. Any tasks at the time will not be cancelled.
+///
+/// Unlike [`Executor`], this type trivially implements both [`Clone`] and [`Copy`].
+///
+/// This type *cannot* be converted back to a `Executor`.
+#[derive(Copy, Clone)]
+#[cfg(feature = "leak")]
```

Comment:
Rather than putting all of this under `cfg(feature = "leak")`, you should move this functionality to another file and then put this feature guard and that file and its import.

Reply:
Done.